src/components/Files/PreUploadModal.tsx

```diff
@@ -305,143 +291,117 @@ export const PreUploadDialog = ({
   }, [folderPath]);
 
   return (
-    <FloatingPortal id="theme-main">
-      {isOpen && (
-        <FloatingOverlay
-          lockScroll
-          className="z-50 flex items-center justify-center bg-blackout p-3"
-          data-floating-overlay
-        >
-          <FloatingFocusManager context={context}>
-            <div
-              className="relative flex max-h-full flex-col gap-4 rounded bg-layer-3 p-6 md:min-w-[425px] md:max-w-[500px]"
-              ref={refs.setFloating}
-              {...getFloatingProps()}
-            >
-              <button
-                className="absolute right-2 top-2"
-                onClick={() => onClose(false)}
-              >
-                <IconX className="text-secondary hover:text-accent-primary" />
-              </button>
-              <div className="flex flex-col gap-2 overflow-auto">
-                <div className="flex justify-between">
-                  <h2 id={headingId} className="text-base font-semibold">
-                    {t('Upload from device')}
-                  </h2>
-                </div>
-                <p id={descriptionId}>
-                  {t(
-                    'Max file size up to 512 Mb. Supported types: {{allowedExtensions}}.',
-                    {
-                      allowedExtensions:
-                        allowedExtensions.join(', ') ||
-                        'no available extensions',
-                    },
-                  )}
-                </p>
-
-                <ErrorMessage error={errorMessage} />
-
-                <div className="flex flex-col gap-1">
-                  <div>
-                    <span className="text-xs text-secondary">
-                      {t('Upload to')}
+    <Modal
+      portalId="theme-main"
+      containerClassName="relative flex max-h-full flex-col gap-4 rounded bg-layer-3 p-6 md:min-w-[425px] md:max-w-[500px]"
```

Comment:
1) previous there were `const dismiss = useDismiss(context, { outsidePressEvent: 'mousedown' });`
I suppose we should have the same i.e. `dismissProps={{ outsidePressEvent: 'mousedown' }}`

2) `relative max-h-full rounded bg-layer-3` are already in `Modal`